odoo/addons/base/tests/__init__.py

```diff
@@ -1,47 +1,48 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# # -*- coding: utf-8 -*-
+# # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from . import common
-from . import test_acl
-from . import test_api
-from . import test_base
-from . import test_basecase
-from . import test_cache
-from . import test_db_cursor
-from . import test_expression
-from . import test_float
-from . import test_func
-from . import test_http_case
-from . import test_image
-from . import test_ir_actions
-from . import test_ir_attachment
-from . import test_ir_http
-from . import test_ir_filters
-from . import test_ir_model
-from . import test_ir_sequence
-from . import test_ir_sequence_date_range
-from . import test_ir_default
-from . import test_mail
-from . import test_menu
-from . import test_mimetypes
-from . import test_misc
-from . import test_orm
-from . import test_ormcache
-from . import test_osv
-from . import test_qweb
-from . import test_res_config
-from . import test_res_lang
-from . import test_search
-from . import test_translate
-#import test_uninstall  # loop
-from . import test_user_has_group
-from . import test_views
-from . import test_xmlrpc
-from . import test_res_partner
-from . import test_res_partner_bank
-from . import test_res_users
-from . import test_reports
-from . import test_tests_tags
-from . import test_base_document_layout
-from . import test_form_create
-from . import test_cloc
+# from . import common
+# from . import test_acl
+# from . import test_api
+# from . import test_base
+# from . import test_basecase
+# from . import test_cache
+# from . import test_db_cursor
+# from . import test_expression
+# from . import test_float
+# from . import test_func
+# from . import test_http_case
+# from . import test_image
+# from . import test_ir_actions
+# from . import test_ir_attachment
+# from . import test_ir_http
+# from . import test_ir_filters
+# from . import test_ir_model
+# from . import test_ir_sequence
+# from . import test_ir_sequence_date_range
+# from . import test_ir_default
+# from . import test_mail
+# from . import test_menu
+# from . import test_mimetypes
+# from . import test_misc
+# from . import test_orm
+# from . import test_ormcache
+# from . import test_osv
+# from . import test_qweb
+# from . import test_res_config
+# from . import test_res_lang
+# from . import test_search
+# from . import test_translate
+# #import test_uninstall  # loop
+# from . import test_user_has_group
+# from . import test_views
+# from . import test_xmlrpc
+# from . import test_res_partner
+# from . import test_res_partner_bank
+# from . import test_res_users
+# from . import test_reports
+# from . import test_tests_tags
+# from . import test_base_document_layout
+# from . import test_form_create
+# from . import test_cloc
```

Comment:
Thanks for updating the tests! :+1:  Now you should probably uncomment the other tests, so we can r+ ;-)

PS: FWIW, you can start the server `--test-file=</path/to/file>` to quickly execute a single Python test file without having to comment out the rest